weatherreport.cpp

```diff
@@ -35,14 +25,31 @@ class SensorStub : public IWeatherSensor {
         return 52;
     }
 };
+class SensorStub2 : public WeatherSpace::IWeatherSensor {
```

Comment:
Another way is to make a single SensorStub class, accepting the values in the constructor and returning them in the individual functions. Then you can make more instances of SensorStub, instead of a new class for every situation

Reply:
changed as suggested